connector/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -367,6 +367,20 @@ abstract class AvroSuite
                     "member_myrecord2: struct<f: float>>",
       Seq())
 
+    // SPARK-47904: Test that field name case is preserved.
```

Comment:
Please create a new test case with a proper test prefix because this test method, `SPARK-43333: ...` is already too long.